test/bom.bats

```diff
@@ -1,13 +1,13 @@
 load helpers
 
 function setup_file() {
-  if [ -n "${ZOT_HOST}:${ZOT_PORT}" ]; then
+  if [ -n "${ZOT_HOST}${ZOT_PORT}" ]; then
```

Comment:
what's the reason for removing the colon? it seems strange to have both host and port in the string to check.

Reply:
shellcheck pointed out that using `-n` with a string that includes a literal string will never be false, because `[ -n ":" ]` is still true - so including the colon meant that we always try to do the zot stuff, even if there's no zot host/port configured.

I guess we could also just remove the port and only check `-n "$ZOT_HOST"`, if we wanted to support using a zot without an explicit port. This just seemed like the simplest change that fixed the issue

Reply:
sounds reasonable.  Thanks